internal/controller/etcdcluster_controller_test.go

```diff
@@ -30,55 +29,69 @@ import (
 	operatorv1alpha1 "go.etcd.io/etcd-operator/api/v1alpha1"
 )
 
-var _ = Describe("EtcdCluster Controller", func() {
-	Context("When reconciling a resource", func() {
-		const resourceName = "test-resource"
+func TestEtcdClusterController(t *testing.T) {
+	ctx := context.Background()
```

Comment:
I believe we can replace this context with regular `T` contexts: 
```suggestion
	ctx := t.Context()
```